### DISCUSSION TITLE
Live View on CSV Import Job Status

### DISCUSSION THREAD

**Discussion by avargas94** [burkybang/SuiteAdvanced-Feedback#58] [Ideas]
Are you able to add the Live View to the CSV Import Job Status?

**Comment by burkybang** [comment]
Actually it's already built in to that page! This is the only page where I force enable Live View. There's no button because it's always enabled.

But thank you for making me realize that it's not very obvious. I think in a later version, I'll include the Live View button but default it to enabled when the page loads. That way users can see that it's enabled, and it can optionally be disabled.
